src/pkg/packager/interactive.go

```diff
@@ -23,32 +23,58 @@ import (
 func (p *Packager) confirmAction(userMessage string, sbomViewFiles []string) (confirm bool) {
 
 	message.HorizontalRule()
+	message.Title("Package Configuration", "the package configuration that defines this package")
 	utils.ColorPrintYAML(p.cfg.Pkg)
 
-	// Print the location that the user can view the package SBOMs from
-	if len(sbomViewFiles) > 0 {
-		cwd, _ := os.Getwd()
-		link := pterm.FgWhite.Sprint(pterm.Bold.Sprint(filepath.Join(cwd, config.ZarfSBOMDir, filepath.Base(sbomViewFiles[0]))))
-		msg := fmt.Sprintf("This package has %d artifacts with software bill-of-materials (SBOM) included. You can view them now in the zarf-sbom folder in this directory or to go directly to one, open this in your browser: %s", len(sbomViewFiles), link)
-		message.HorizontalNoteRule()
-		message.Note(msg)
-		message.Note(" * This directory will be removed after package deployment.")
-	}
-
 	// Print any potential breaking changes (if this is a Deploy confirm) between this CLI version and the deployed init package
 	if userMessage == "Deploy" {
+		// Print the location that the user can view the package SBOMs from
+		message.HorizontalRule()
+		message.Title("Software Bill of Materials", "an inventory of all software contained in this package")
+
+		if len(sbomViewFiles) > 0 {
+			cwd, _ := os.Getwd()
+			link := pterm.FgLightCyan.Sprint(pterm.Bold.Sprint(filepath.Join(cwd, config.ZarfSBOMDir, filepath.Base(sbomViewFiles[0]))))
+			inspect := pterm.BgBlack.Sprint(pterm.FgWhite.Sprint(pterm.Bold.Sprintf("$ zarf package inspect %s", p.cfg.PkgSourcePath)))
+
+			artifactMsg := pterm.Bold.Sprintf("%d artifacts", len(sbomViewFiles)) + " to be reviewed. These are"
+			if len(sbomViewFiles) == 1 {
+				artifactMsg = pterm.Bold.Sprintf("%d artifact", len(sbomViewFiles)) + " to be reviewed. This is"
+			}
+
+			msg := fmt.Sprintf("This package has %s available in a temporary '%s' folder in this directory and will be removed upon deployment.\n", artifactMsg, pterm.Bold.Sprint("zarf-sbom"))
+			viewNow := fmt.Sprintf("\n- View SBOMs %s by navigating to the '%s' folder or copying this link into a browser:\n%s", pterm.Bold.Sprint("now"), pterm.Bold.Sprint("zarf-sbom"), link)
+			viewLater := fmt.Sprintf("\n- View SBOMs %s deployment with this command:\n%s", pterm.Bold.Sprint("after"), inspect)
+
+			message.Note(msg)
+			pterm.Println(viewNow)
+			pterm.Println(viewLater)
+		} else {
+			message.Warn("This package does NOT contain an SBOM.  If you require an SBOM, please contact the creator of this package to request a version that includes an SBOM.")
```

Comment:
I feel like we should only log this as a warning if there is something in the package that is SBOM-able